packages/react-magma-dom/src/components/Pagination/SimplePagination.test.js

```diff
@@ -0,0 +1,178 @@
+    const { getByLabelText } = render(
+      <Pagination simple count={4} defaultPage={2} testId={testId} />
+    );
+    let previousButton = getByLabelText('Previous Page');
+
+    fireEvent.click(previousButton);
+
+    expect(previousButton).toHaveStyleRule(
+      'color',
+      transparentize(0.4, magma.colors.neutral500)
+    );
+  });
+
+  it('Should disable the next button upon selecting the last page', () => {
+    const { getByLabelText } = render(
+      <Pagination simple count={4} defaultPage={3} testId={testId} />
+    );
+    let nextButton = getByLabelText('Next Page');
+
+    fireEvent.click(nextButton);
+
+    expect(nextButton).toHaveStyleRule(
+      'color',
+      transparentize(0.4, magma.colors.neutral500)
+    );
+  });
+
+  it('Should render a large variant of the pagination button', () => {
+    const { container } = render(
+      <Pagination count={4} simple size={'large'} />
+    );
+    const previousButton = container.querySelector('button');
+
+    expect(previousButton).toHaveStyleRule(
+      'height',
+      magma.spaceScale.spacing11
+    );
+  });
+
+  it('Should change the active page when an option is selected', () => {
+    const { getByTestId, getAllByTestId } = render(
+      <Pagination simple count={4} testId={testId} />
+    );
+    fireEvent.change(getByTestId(`${testId}-select`), { target: { value: 2 } });
+    let options = getAllByTestId(`${testId}-option`);
+    expect(options[0].selected).toBeFalsy();
+    expect(options[1].selected).toBeTruthy();
+    expect(options[2].selected).toBeFalsy();
+  });
+
+  it('Should change the active page when clicking the previous button', () => {
+    const { getByLabelText, getAllByTestId } = render(
+      <Pagination simple count={4} defaultPage={4} testId={testId} />
+    );
+    let previousButton = getByLabelText('Previous Page');
+
+    fireEvent.click(previousButton);
+    let options = getAllByTestId(`${testId}-option`);
+    expect(options[0].selected).toBeFalsy();
+    expect(options[1].selected).toBeFalsy();
+    expect(options[2].selected).toBeTruthy();
+    expect(options[3].selected).toBeFalsy();
+  });
+
+  it('Should change the active page when clicking the next button', () => {
+    const { getByLabelText, getAllByTestId } = render(
+      <Pagination simple count={4} defaultPage={2} testId={testId} />
+    );
+    let nextButton = getByLabelText('Next Page');
+
+    fireEvent.click(nextButton);
+    let options = getAllByTestId(`${testId}-option`);
+    expect(options[0].selected).toBeFalsy();
+    expect(options[1].selected).toBeFalsy();
+    expect(options[2].selected).toBeTruthy();
+    expect(options[3].selected).toBeFalsy();
+  });
+
+  it('Does not violate accessibility standards', () => {
+    const { container } = render(<Pagination simple count={4} />);
+
+    return axe(container.innerHTML).then(result => {
+      return expect(result).toHaveNoViolations();
+    });
+  });
+});
```

Comment:
Lets add tests for edge cases:
- what if defaultPage or count is 0? -> what do we expect to happen?
- what if defaultPage or count is 1? 

What about coverage for the other Pagination props: hidePreviousButton, numberOfAdjacentPages, etc etc.

Reply:
> Lets add tests for edge cases:
> 
> * what if defaultPage or count is 0? -> what do we expect to happen?
> * what if defaultPage or count is 1?
> 
> What about coverage for the other Pagination props: hidePreviousButton, numberOfAdjacentPages, etc etc.

Looks like the tests for the extra props have been addressed, but would still love to see more unit tests from the first two points, especially since you are now handling the singular/plural "pages" label change

Reply:
Oooh, and tests that when a page gets changed (1. from previous button 2. from next button 3. from dropdown), the `onPageChange` function gets called